### PR TITLE
feat(libflux/flux): exposed function to get docs in WASM

### PR DESCRIPTION
**Pull request by BarWeiner1**
### Done checklist
- [ ] exposed function to get JS values for flux documentation given a flux identifier
- [ ] Test cases written

### CODE REVIEW

**Comment by scbrickley** [comment]
Because I helped work on this, I don't think it's appropriate for me to give it a review. Another set of eyes would be helpful here.

Also, even after we get a thumbs up from someone on the Flux team, we should wait until we've gotten someone from the UI team to sign off on this before merging.

**Review by fchikwekwe — APPROVED** [review]
Good to see tests passing. This looks fine to me after the changes. Just make sure to squash all these commits when you merge.

File: libflux/flux/src/wasm/mod.rs
Comment (kristinarobinson) [inline libflux/flux/src/wasm/mod.rs:95]: I think you meant in this part of the text:
`<code>file</code> if the file path of the CSV file to query.`
to be 
`<code>file</code> **is** the file path of the CSV file to query.`

Comment (BarWeiner1) [reply libflux/flux/src/wasm/mod.rs:95]: This is a problem with our hand written documentation of the csv.flux file not with this issue. Let me merge this through to avoid future problems with CI and rebasing, and then I will put up a patch to fix the csv.flux file and this test.

Comment (BarWeiner1) [reply libflux/flux/src/wasm/mod.rs:95]: This is a problem with our hand written documentation of the csv.flux file not with this issue. Let me merge this through to avoid future problems with CI and rebasing, and then I will put up a patch to fix the csv.flux file and this test.

Comment (BarWeiner1) [reply libflux/flux/src/wasm/mod.rs:95]: Screen Shot 2021-07-19 at 11 18 10 AM

**Review by kristinarobinson — APPROVED** [review]

**Pull request merged**
